src/components/screens/PointOfInterest.js

```diff
@@ -55,6 +56,8 @@ export const PointOfInterest = ({ data, hideMap, navigation }) => {
         rootRouteName
       }
     });
+
+  const logo = dataProvider && dataProvider.logo && dataProvider.logo.url;
```

Comment:
this was intended copy & paste from tours instead of refactoring to optional chaining at the same time?

Reply:
i wanted to touch as little as possible, as both components are really similar and should be refactored to reduce code duplication.

this way it remains easier to see the similarities between the files when it comes to that.

i did not want to touch both files.